test/unit_tests/apps/owning-a-home/js/dropdown-utils-spec.js

```diff
@@ -95,8 +95,10 @@ describe( 'Dropdown utils', () => {
   } );
 
   it( 'should select several select elements', () => {
+    /* eslint-disable max-line */
```

Comment:
`max-len`?

Reply:
That was a typo! I change max-line to max-len